packages/story-editor/src/app/currentUser/currentUserProvider.tsx

```diff
@@ -49,13 +57,18 @@ function CurrentUserProvider({ children }) {
   }, [currentUser, getCurrentUser]);
 
   const updateCurrentUser = useCallback(
-    (data) =>
-      _updateCurrentUser ? _updateCurrentUser(data).then(setCurrentUser) : null,
+    (data: updateCurrentUserProps) =>
+      _updateCurrentUser
+        ? void _updateCurrentUser(data).then(setCurrentUser)
+        : null,
     [_updateCurrentUser]
   );
 
   const toggleWebStoriesMediaOptimization = useCallback(() => {
-    return updateCurrentUser({
+    if (!currentUser) {
+      return;
+    }
+    void updateCurrentUser({
```

Comment:
Ditto here about `void`, `toggleWebStoriesMediaOptimization` should return the Promise from `updateCurrentUser`

Reply:
Fixed.